formats/developers/sample-files.txt

```diff
@@ -1,115 +1,14 @@
-   contains 11 images. This file shows how these images can be
-   attached to wells within a plate.
-
--  ``6x4y1z1t1c8b-swatch-2s2p4w2ws.ome`` 
-   contains 22 images. This file shows how two plates and two
-   screens can be linked together.
-
--  ``6x4y1z1t1c8b-swatch-instrument.ome`` 
-   has a single image with an instrument using lights sources,
-   detectors, objectives and filters (most of the other examples do not 
-   contain any instrument data).
-
--  ``6x4y1z1t1c8b-swatch-tagged.ome`` 
-   has a single image with two tags and a tagset
-   attached to it using annotations.
-
--  ``6x4y1z1t1c8b-swatch-xmlannotation-svg.ome`` 
-   has a single image. The image has a block of XML data
-   attached to it using annotations. The XML data is SVG in this
-   example but could be any well formed XML.
-
--  ``6x4y1z1t1c8b-swatch.ome`` 
-   has a single image. This file is the one on which the other
-   files are based.
-
--  ``6x4y1z1t3c8b-swatch-ROI.ome`` 
-   has a single image, but that image has 3 channels. There are
-   3 ROIs defined, two on only a single channel and one that applies to
-   all the channels.
-
--  ``6x4y1z1t3c8b-swatch.ome`` 
-   has a single image, but that image has 3 channels.
-    
--  ``18x24y1z5t1c8b-text.ome`` 
-   has a single image. The image has pixels stored for 5
-   timepoints.
-
--  ``18x24y1z5t2c8b-text.ome`` 
-   has a single image. The image has pixels stored for 5
-   timepoints. Each of these timepoints has 2 channels to give a total
-   of 10 bit planes.
-
--  ``18x24y5z1t1c8b-text.ome`` 
-   has a single image. The image has pixels stored for 5
-   z-sections.
-
--  ``18x24y5z1t2c8b-text.ome`` 
-   has a single image. The image has pixels stored for 5
-   z-sections. Each of these z-sections has 2 channels to give a total
-   of 10 bit planes.
-
--  ``18x24y5z5t1c8b-metadata.ome`` 
-   defines a single image. The image is defined as having pixels
-   for 5 z-sections. Each of these z-sections has 5 timepoints to give
-   a total of 25 bit planes. This is a metadata-only file. It only has
-   the metadata defining the image, not any of the pixel data.
-
--  ``18x24y5z5t1c8b-text.ome`` 
-   has a single image. The image has pixels stored for 5
-   z-sections. Each of these z-sections has 5 timepoints to give a
-   total of 25 bit planes.
-
--  ``18x24y5z5t2c8b-text.ome`` 
-   has a single image. The image has pixels stored for 5
-   z-sections. Each of these z-sections has 5 timepoints. Each of these
-   timepoints has 2 channels to give a total of 50 bit planes.
-
-Other OME-TIFF files
---------------------
+date e.g. folder :image_downloads:`OME-TIFF/2015-01/` has samples using schema
+version :schema_plone:`OME/2015-01`.
 
-See the :doc:`/ome-tiff/data` section.
+See the :doc:`/ome-tiff/data` section for more information about the sample
+TIFF files. The OME-XML files contained in the :image_downloads:`OME-XML`
+folder have all been renamed so they should be self-explanatory.
```

Comment:
maybe note in some later release to adjust this so that it doesn't look like they were still recently renamed